config/README.md

```diff
@@ -0,0 +1,100 @@
+Firebase Remote Config Quickstart
+==============================
+
+The Firebase Remote Config Web quickstart app demonstrates using Remote
+Config to define user-facing text in a web app.
+
+Introduction
+------------
+
+This is a simple example of using Remote Config to override in-app default
+values by defining service-side parameter values in the Firebase console. This
```

Comment:
We switched over to just saying "server-side" or "backend" per Mayank's request as of c/279341337.

Reply:
done!